src/Microsoft.Azure.WebJobs.Storage/Blob/StorageBlobClient.cs

```diff
@@ -58,6 +58,16 @@ public Task<ServiceProperties> GetServicePropertiesAsync(CancellationToken cance
             BlobListingDetails blobListingDetails, int? maxResults, BlobContinuationToken currentToken,
             BlobRequestOptions options, OperationContext operationContext, CancellationToken cancellationToken)
         {
+            // Add WebJobs to user agent to make it easier to collect usage information from Storage telemetry
+            if (operationContext == null)
+            {
+                operationContext = new OperationContext();
+            }
+            operationContext.SendingRequest += (sender, e) =>
+            {
+                e.Request.UserAgent += "-WebJobsBlob";
```

Comment:
Can we use a single agent identifier like AzureWebJobs? Not sure we need to have Blob/Queue variants.